docs-v2/content/en/docs/pipeline-stages/init.md

```diff
@@ -17,7 +17,8 @@ and [deploy](#deploy-config-initialization) config.
 
 1. [Docker]({{<relref "/docs/pipeline-stages/builders/docker">}})
 2. [Jib]({{<relref "/docs/pipeline-stages/builders/jib">}})
-2. [Buildpacks]({{<relref "/docs/pipeline-stages/builders/buildpacks">}})
+3. [Ko]({{<relref "/docs/pipeline-stages/builders/ko">}})
+4. [Buildpacks]({{<relref "/docs/pipeline-stages/builders/buildpacks">}})
```

Comment:
Ko is preferred over buildpacks right?
if an artifact with `go.mod` is present, init will configure ko artifact over buildpacks?

Reply:
If a user runs `skaffold init` and a `go.mod` file is present, the user will get a prompt to select Buildpacks or Ko (the list of builder options) is sorted alphabetically:

```shell
$ skaffold init
? Choose the builder to build image skaffold-ko  [Use arrows to move, type to filter]
> Buildpacks (go.mod)
  Ko (go.mod)
  None (image not built from these sources)
```

If the user adds the `--force` flag, then init will select the Ko builder in preference over Buildpacks:

```shell
$ skaffold init --force
Configuration skaffold.yaml was written
You can now run [skaffold build] to build the artifacts
or [skaffold run] to build and deploy
or [skaffold dev] to enter development mode, with auto-redeploy
```

```shell
$ cat skaffold.yaml
apiVersion: skaffold/v3alpha1
kind: Config
metadata:
  name: ko
build:
  artifacts:
  - image: skaffold-ko
    ko:
      dependencies:
        paths:
        - '**/*.go'
        - go.*
      dir: .
manifests:
  rawYaml:
  - k8s/example.yaml
```